acceptance/tests/apply_plan_winrm.rb

```diff
@@ -21,10 +21,11 @@
     scp_to(bolt, File.join(fixtures, 'example_apply'), "#{dir}/modules/example_apply")
   end
 
-  bolt_command = "bolt plan run example_apply filepath=#{filepath} nodes=winrm_nodes"
+  bolt_command = "bolt plan run example_apply filepath=#{filepath}"
   flags = {
     '--modulepath' => modulepath(File.join(dir, 'modules')),
-    '--format' => 'json'
+    '--format' => 'json',
+    '-t' => winrm_nodes.to_s
```

Comment:
Why do we use the `winrm_nodes` variable here and not the string `"winrm_nodes"` as we were using before?

Reply:
Oh my gosh I thought I was going insane. Didn't even think about winrm_nodes vs "winrm_nodes" :woman_facepalming: